dev/com.ibm.ws.classloading.bells_fat/fat/src/com/ibm/ws/classloading/BellFatTest.java

```diff
@@ -101,9 +106,21 @@ static void stopServer() {
      */
     @Test
     @AllowedFFDC({INSTANTIATION_EXCEPTION, CLASS_NOT_FOUND_EXCEPTION, NO_CLASSDEF_FOUND_EXCEPTION, EXCEPTION} )
+    @SuppressWarnings("serial")
     public void testNoEntry() throws Exception {
 
-        assertNotNull("Should warn that no services were found to export from the 'testNoEntry' library.", server.waitForStringInLog(".*CWWKL0055W: .*testNoEntry"));
+        Map<String,String> sysProps = new HashMap<String,String>(3){};
+        sysProps.put(BETA_EDITION_JVM_OPTION, Boolean.FALSE.toString());
+        try {
+            setSysProps(server, sysProps);
+            server.startServer();
+
+            assertNotNull("Should warn that no services were found to export from the 'testNoEntry' library.",
+                    server.waitForStringInLog(".*CWWKL0055W: .*testNoEntry"));
+        } finally {
+            stopServer();
+            removeSysProps(server, sysProps);
```

Comment:
Can you removeSysProps before stopServer here?  Otherwise you risk no calling removeSysProps if stopServer fails.

Reply:
The test works as intended when `removeSysProps()` is called before `stopServer()`. The call to `stopServer()` is vestigial as the method is also called by `@After afterTest()`. So, I removed the call to `stopServer()` from all tests that manage server system props, except test `BellPropertiesTest.testPropertiesInvalid()`. BTW, this change was introduced to ensure the test rewrites the jvm.options file before `bells_fat_server` is started for the first time.